packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/backups.routing.js

```diff
@@ -0,0 +1,64 @@
+import find from 'lodash/find';
+import get from 'lodash/get';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.storages.databases.dashboard.backups',
+    {
+      url: '/backups',
+      cache: false,
+      views: {
+        databaseView: 'ovhManagerPciStoragesDatabaseBackupsComponent',
+      },
+      resolve: {
+        /* @ngInject */
+        backupList: (database, DatabaseService, projectId) =>
+          DatabaseService.getBackups(projectId, database.engine, database.id),
+        goBackToBackups: /* @ngInject */ ($state, CucCloudMessage) => (
+          message = false,
+          type = 'success',
+        ) => {
+          const reload = message && type === 'success';
+          const state =
+            'pci.projects.project.storages.databases.dashboard.backups';
+          const promise = $state.go(state, {}, { reload });
+          if (message) {
+            promise.then(() => {
+              CucCloudMessage[type](message, state);
+            });
+          }
+          return promise;
+        },
+        goToCreateBackup: /* @ngInject */ ($state) => () =>
+          $state.go(
+            'pci.projects.project.storages.databases.dashboard.backups.create',
+          ),
+        goToRestore: /* @ngInject */ ($state) => (backupInstance) =>
+          $state.go(
+            'pci.projects.project.storages.databases.dashboard.backups.restore',
+            {
+              backupInstance,
+            },
+          ),
+        refreshBackups: /* @ngInject */ ($state) => () => {
+          return $state.reload();
+        },
+        catalog: () => ({
+          addons: [],
+        }),
```

Comment:
Missing API call ?

Reply:
This code was not required and has been removed now. Thanks